Formula/sic.rb

```diff
@@ -4,7 +4,7 @@ class Sic < Formula
   url "https://dl.suckless.org/tools/sic-1.2.tar.gz"
   sha256 "ac07f905995e13ba2c43912d7a035fbbe78a628d7ba1c256f4ca1372fb565185"
   license "MIT"
-  head "https://git.suckless.org/sic.git", branch: "master"
+  head "https://git.suckless.org/sic", using: :git
```

Comment:
I think we want to keep the `branch: "master"` for this still.

Also, needing to specify `using: :git` seems like a bug in `brew`? Or does `https://git.suckless.org/sic.git` no longer resolve correctly?

Reply:
`dmenu`, `dwm` & `ii` use the same server.
```
$ git clone https://git.suckless.org/sic.git
Cloning into 'sic'...
fatal: repository 'https://git.suckless.org/sic.git/' not found
$ git clone https://git.suckless.org/sic
Cloning into 'sic'...
remote: Enumerating objects: 452, done.
remote: Counting objects: 100% (452/452), done.
remote: Compressing objects: 100% (302/302), done.
remote: Total 452 (delta 203), reused 367 (delta 147), pack-reused 0
Receiving objects: 100% (452/452), 174.83 KiB | 309.00 KiB/s, done.
Resolving deltas: 100% (203/203), done.
```
The DownloadStrategyDetector only checks for these patterns for git:
```
    when %r{^https?://.+\.git$},
         %r{^git://},
         %r{^https?://git\.sr\.ht/[^/]+/[^/]+$}
      GitDownloadStrategy
```
Having it assume that a URL should use git if its domain starts with "git." might be an idea, though.

Reply:
It's an idea, but one that doesn't seem to apply universally, sadly. (e.g. `opencascade`)